pylint_odoo/test_repo/test_module/except_pass.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""Test Except Pass usage"""
+
+
+class TestExceptPass(object):
+    """Test Except Pass class """
+
+    def test_method(self):
+        """Test method """
+        try:
+            raise Exception('Exception')
+        except Exception:
+            pass
+
+    def test_2_method(self):
+        """Test 2 method """
+        try:
+            raise Exception('Exception')
+        except Exception:
+            print('Exception')
+            pass
+
+    def test_3_method(self):
+        """Test 3 method """
+        try:
+            raise Exception('Exception')
+        except Exception as exception:
+            pass
+        if exception:
+            pass
+
+    def test_4_method(self):
+        try:
+            raise Exception('Exception')
+        except Exception, UserError:
+            pass
+        if UserError:
+            pass
+
+    def test_5_method(self):
+        try:
+            raise Exception('Exception')
+        except (Exception, IndexError) as exception:
+            pass
+        if exception:
+            pass
+
+    def test_6_method(self):
+        try:
+            raise Exception('Exception')
+        except (Exception, IndexError, NameError), exception:
+            pass
+        except Exception:
```

Comment:
Add a comment where the message should be emitted:
`except Exception:  # except-pass`